downstairs/src/lib.rs

```diff
@@ -1187,19 +1276,35 @@ impl Downstairs {
         ds_id: u64,
     ) -> Result<()> {
         match m {
-            Message::FlushAck(_, _, _) => {
+            Message::FlushAck {
+                upstairs_id: _,
```

Comment:
Yeah, otherwise the compiler complains:

```
error: expected identifier, found reserved identifier `_`
    --> downstairs/src/lib.rs:1280:17
     |
1280 |                 _,
     |                 ^ expected identifier, found reserved identifier
```

Reply:
But using `{ .. }` works! Changed in 03b8395